### DISCUSSION TITLE
Prevent Monster Heal, Slain Monsters Rest In Peace

### DISCUSSION THREAD

**Discussion by TrazhCant** [blizzhackers/pickits#10] [Q&A about pickit codes]
Is it possible to check for "Prevent Monster Heal" or "Slain Monsters Rest In Peace"?  I have scoured Google and all documentation here (which is fantastic, btw), but I was unable to determine how these might be checked.  The vast majority of item stats are quantified in whole numbers, but these 2 stats do not appear to have any number value associated with them.  Would they possibly be checked with true/false values instead of numbers?  Thanks in advance.

**Comment by mf022** [comment]
> "Prevent Monster Heal"

https://github.com/blizzhackers/kolbot/blob/master/d2bs/kolbot/libs/NTItemAlias.dbl/#L1181

> "Slain Monsters Rest In Peace"

https://github.com/blizzhackers/kolbot/blob/master/d2bs/kolbot/libs/NTItemAlias.dbl/#L1172

So you should try to use in {stats} part of the pickit line:
 [itemrestinpeace] && ...
 [itempreventheal] && ...
